lessons/Lesson-4-Ruby-Part-2.md

```diff
@@ -116,47 +157,52 @@ end
 RectangleOne.report
 RectangleOne.report
 
-rect1 = RectangleOne.new(5,4)
-rect2 = RectangleOne.new(6,3)
-  
+rect1 = RectangleOne.new(5, 4)
+rect2 = RectangleOne.new(6, 3)
+
 puts rect1.height
 puts rect1.area
 puts rect2.area
-  
+
 rect3 = RectangleTwo.new(7, 2)
 puts rect3.area
 puts rect3.width
 rect3.width=4
 puts rect3.width
 puts rect3.height # throws an error
-
 ```
+
 Ok, deep breath, let's see what are special about Ruby classes.
 
-First, you often have an initializer.  This is a method of the class that is called when you create an instance of the class.  To create an instance of the class, you call the new method on the class itself.  Second, you typically have instance variables.  These are values stored with that instance.  Each instance has a different set of values for instance variables.  The instance variables always start with @.  That's the only way to keep values associated with the instance.  The class may have one or more instance methods, such as area in the examples above.  A method is a function associated with an instance of a class.  If you run the program above (you will get an error to be explained shortly), you note that rect2 has a different area than rect1.  They are separate instances of the same class.
+First, you often have an initializer.  This is a method of the class that is called when you create an instance of the class.  To create an instance of the class, you call the `new` method on the class itself.  Second, you typically have instance variables.  These are values stored with that instance.  Each instance has a different set of values for instance variables.  The instance variables always start with `@`.  That's the only way to keep values associated with the instance.  The class may have one or more instance methods, such as `area` in the examples above.  If you run the program above (you will get an error to be explained shortly), you note that `rect2` has a different area than `rect1`.  They are separate instances of the same class.
```

Comment:
No "function" in Ruby; see [other comment](https://github.com/Code-the-Dream-School/ROR-lesson-3-and-4/pull/6/files#r1605884405)